buildspecs/linux_ppc-64.spec

```diff
@@ -58,7 +58,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<property name="graph_req.aux0" value=""/>
 		<property name="graph_req.aux1" value=""/>
 		<property name="graph_req.build" value="build:java8sr5fp27+"/>
-		<property name="graph_req.build2" value="build:java8sr5fp27+"/>
+		<property name="graph_req.build2" value="build:java8-xlc13"/>
```

Comment:
Sorry, I don't know how this value is used, but I assume it's correct.

Reply:
It is an infrastructure setting for automated builds.

Reply:
@pshipton do we keep `java8-xlc13` machine label:  or do you want to rename it?

Reply:
We should rename it to java8sr6fp5+, but before you make this change to the machine capabilities, we should review the rest of the changes.

Reply:
I renamed java8-xlc13 to java8sr6fp5+.